### PR TITLE
Cleaning the htmlcov directory

### PR DESCRIPTION
**Pull request by scbedd**
The `code coverage` tab in DevOps when all `azure-*` packages are built NEVER displays. It just spins forever.

The reason for this is that `coverage html` generates an html page for every single test. This means that the the build artifacts for the cobertura report contains about 6k html files that prevent the UI from ever loading (huge download).

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: scripts/devops_tasks/inline_css_for_cobertura.py [hunk 1]
```diff
@@ -38,9 +39,36 @@ def embed_css_in_html_file(html_file, css_dir):
...
+    # write final results
+    with io.open(html_file, 'w', encoding='utf-8') as f:
+        f.write(__str_version_handler(soup))
 
 if __name__ == '__main__':
     for file in os.listdir(COVERAGE_REPORT_DIR):
         if file.endswith(".html"):
-            print("Embedding CSS in {}".format(file))
-            embed_css_in_html_file(os.path.join(COVERAGE_REPORT_DIR, file), COVERAGE_REPORT_DIR)
+            if 'index.html' not in file:
```
Comment (bsiegel) [inline scripts/devops_tasks/inline_css_for_cobertura.py]: I would probably invert these conditions, and use os.path to decompose the file path:

```
name, ext = os.path.splitext(os.path.basename(file.lower()))
if ext == ".html":
    if name == "index":
        ...
    else:
        ...
```

Comment (scbedd) [reply scripts/devops_tasks/inline_css_for_cobertura.py]: Cheers.

**Pull request merged**